src/AuctionHouseBotAuctionHouseScript.cpp

```diff
@@ -1,4 +1,9 @@
+/*
+ * Copyright (C) 2016+ AzerothCore <www.azerothcore.org>, released under GNU AGPL v3 license: https://github.com/azerothcore/azerothcore-wotlk/blob/master/LICENSE-AGPL3
```

Comment:
file not found, let's put `https://github.com/azerothcore/azerothcore-wotlk/blob/master/LICENSE`

Reply:
Done as requested.